CONTRIBUTING.md

```diff
@@ -0,0 +1,28 @@
+# Contributing
+
+We follow [Ansible Code of Conduct](https://docs.ansible.com/ansible/latest/community/code_of_conduct.html) in all our contributions and interactions within this repository.
+
+If you are a committer, also refer to the [collection's committer guidelines](https://github.com/ansible-collections/community.general/blob/main/commit-rights.md).
+
+## Issue tracker
+
+Whether you are looking for an opportunity to contribute or you found a bug and already know how to solve it, please go to the [issue tracker](https://github.com/ansible-collections/community.general/issues).
+There you can find feature ideas to implement, reports about bugs to solve, or submit an issue to discuss your idea before implementing it which can help choose a right direction at the beginning of your work and potentially save a lot of time and effort.
+Also somebody may already have started discussing or working on implementing the same or a similar idea,
+so you can cooperate to create a better solution together.
```

Comment:
```suggestion
so you can cooperate to create a better solution together.

* If you are interested in starting with an easy issue, look for [issues with an `easyfix` label](https://github.com/ansible-collections/community.general/labels/easyfix).
* Often issues that are waiting for contributors to pick up have [the `waiting_on_contributor` label](https://github.com/ansible-collections/community.general/labels/waiting_on_contributor).
```

Reply:
This is great, but how do you think we can show the difference between these two label?
I mean something like:
Often issues that need a little more work have [`waiting_on_contributor` label] ...

Reply:
I'd say that `waiting_on_contributor` usually requires some experience, with Ansible, the module/plugin codebase, and/or programming in general. It basically says "the maintainers don't want to implement this, but think it is useful, so if someone else wants to implement this...".

Reply:
is it worth to also have and mention `documentation` and `help_wanted` labels?

Reply:
`waiting_on_contributor` is Ansible's name for `help_wanted` :) The documentation label would be `docs`. Link would be https://github.com/ansible-collections/community.general/issues?q=type%3Aissue+is%3Aopen+label%3Adocs+